src/encoded/static/components/blocks/richtext.js

```diff
@@ -25,7 +25,7 @@ export default class RichTextBlockView extends React.Component {
         // Polyfill for 'closest' for IE
         const elementPrototype = window.Element.prototype;
         if (!elementPrototype.closest) {
-            elementPrototype.closest = (s) => {
+            elementPrototype.closest = function closest(s) {
```

Comment:
the this keyword (function context) needs to come from closest-function and not the parent function. With arrow function, this-keywords comes from the parent function which is undesired. For some reason, only IE recognizes this behaviors correctly.